src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.SpecialTypes.cs

```diff
@@ -154,56 +154,64 @@ public static object[][] DefaultValues
+                ["bool", "true", true, true],
+                ["System.Threading.CancellationToken", "default", default(CancellationToken), false],
+                ["Todo?", "default", default(Todo), false],
+                ["char", "\'a\'", 'a', true],
+                ["int", "default", 0, true],
+                ["int", "1234", 1234, true],
+                ["int", "1234 * 4", 1234 * 4, true],
+                ["double", "1.0", 1.0, true],
+                ["double", "double.NaN", double.NaN, true],
+                ["double", "double.PositiveInfinity", double.PositiveInfinity, true],
+                ["double", "double.NegativeInfinity", double.NegativeInfinity, true],
+                ["double", "double.E", double.E, true],
+                ["double", "double.Epsilon", double.Epsilon, true],
+                ["double", "double.NegativeZero", double.NegativeZero, true],
+                ["double", "double.MaxValue", double.MaxValue, true],
+                ["double", "double.MinValue", double.MinValue, true],
+                ["double", "double.Pi", double.Pi, true],
+                ["double", "double.Tau", double.Tau, true],
+                ["float", "float.NaN", float.NaN, true],
+                ["float", "float.PositiveInfinity", float.PositiveInfinity, true],
+                ["float", "float.NegativeInfinity", float.NegativeInfinity, true],
+                ["float", "float.E", float.E, true],
+                ["float", "float.Epsilon", float.Epsilon, true],
+                ["float", "float.NegativeZero", float.NegativeZero, true],
+                ["float", "float.MaxValue", float.MaxValue, true],
+                ["float", "float.MinValue", float.MinValue, true],
+                ["float", "float.Pi", float.Pi, true],
+                ["float", "float.Tau", float.Tau, true],
+                ["decimal", "decimal.MaxValue", decimal.MaxValue, true],
+                ["decimal", "decimal.MinusOne", decimal.MinusOne, true],
+                ["decimal", "decimal.MinValue", decimal.MinValue, true],
+                ["decimal", "decimal.One", decimal.One, true],
+                ["decimal", "decimal.Zero", decimal.Zero, true],
+                ["long", "long.MaxValue", long.MaxValue, true],
+                ["long", "long.MinValue", long.MinValue, true],
+                ["short", "short.MaxValue", short.MaxValue, true],
+                ["short", "short.MinValue", short.MinValue, true],
+                ["ulong", "ulong.MaxValue", ulong.MaxValue, true],
+                ["ulong", "ulong.MinValue", ulong.MinValue, true],
+                ["ushort", "ushort.MaxValue", ushort.MaxValue, true],
+                ["ushort", "ushort.MinValue", ushort.MinValue, true],
+                ["TodoStatus", "TodoStatus.Done", TodoStatus.Done, true],
+                ["TodoStatus", "TodoStatus.InProgress", TodoStatus.InProgress, true],
+                ["TodoStatus", "TodoStatus.NotDone", TodoStatus.NotDone, true],
```

Comment:
```suggestion
                ["TodoStatus", "TodoStatus.NotDone", TodoStatus.NotDone, true],
                ["TodoStatus", "1", 1, true],
```
Enum types don't match